assets/onboarding/data/actions.js

```diff
@@ -142,6 +143,7 @@ export function* submitStep( step, stepData, { onSuccess, onError } ) {
 			data: stepData,
 		} );
 		yield successSubmit();
+		yield applyStepData( step, stepData );
```

Comment:
Maybe a middleware could be a good idea to use here? (I'm not sure how to use middleware with `@wordpress/data`, but I imagine it's similar to redux.

Reply:
I did some reading up on this, and feels like the controls would be better suited for state-specific side effect like this. A middleware would be for a more generic extension of the entire store. WDYT?

Reply:
I usually like middleware to isolate this type of logic. But it's ok for me to use the controls. I'm wondering if would make sense to tie this name to the log. Maybe `toggleLog`. Would it make sense?

Reply:
I intended this to be generic for any side-effects triggered by step data - since the submitting itself is generic for all steps. We can also just move all this out of the store and call `logEvent.enable()` right from the Welcome step, but this seemed more robust.